Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10
+
+RUN apt update
```

Comment:
Combine RUN commands: You can combine multiple RUN commands into a single command to reduce the number of layers in your Docker image.